src/widget.tsx

```diff
@@ -72,10 +73,24 @@ const App = (props: any): JSX.Element => {
     setSelectedJupyterItems(selectedJupyterItemsTemp);
   };
 
+  const handleLogout = (event) => {
+    event.preventDefault();
+    window.open(getBaseURL('globus-jupyterlab/logout'), 'Logout of Globus', 'height=600,width=800').focus();
+    window.location.reload();
+  };
   return (
     <div className='container pt-5'>
+      <div className='row'>
+        <div className='col-8'>
+          <a href='#' onClick={handleLogout}>
+            <i className='fa-solid fa-arrow-right-from-bracket'></i> Logout of Globus
+          </a>
+
+          <hr className='mb-3 mt-3' />
+        </div>
+      </div>
```

Comment:
Placement of this is a bit odd right above the transfer window. I think your earlier idea to put the option in a file menu was much better.